crates/shim/src/args.rs

```diff
@@ -44,6 +44,7 @@ pub struct Flags {
 }
 
 /// Parses command line arguments passed to the shim.
+#[cfg_attr(feature = "tracing", tracing::instrument(parent = tracing::Span::current(), skip_all, level = "Info"))]
```

Comment:
If you do not specify the `parent` argument, the macro will by default use the currently active span as the parent. Seems like `parent = tracing::Span::current()` is not needed?

Reply:
Agreed. Will remove `parent = ...`. 

For `level = ...`, I actually prefer to explictly state `"Info"` since it's clearer what level of tracing are we collecting here.